client/components/lists/listHeader.jade

```diff
@@ -11,11 +11,11 @@ template(name="listHeader")
         = title
         if wipLimit.enabled
          |&nbsp;(
-         span(class="{{#if reachedWipLimit}}highlight{{/if}}") {{cards.count}}
+         span(class="{{#if reachedWipLimit}}highlight{{/if}}") {{cardsCount}}
```

Comment:
Thinking of this more, the WIP limit is set per list, so here, we should probably keep the full count of cards, not just the ones that are in the current swimlane. I'll drop this change in the next version.